integration-tests/src/test/java/com/github/dbmdz/flusswerk/integration/RabbitUtil.java

```diff
@@ -98,7 +98,8 @@ public void purgeQueues() {
 
   public List<Queue> allQueues() {
     Stream<String> queueNames = routing.getIncoming().stream();
-    queueNames = Stream.concat(queueNames, routing.getOutgoing().values().stream());
+    queueNames =
+        Stream.concat(queueNames, routing.getOutgoing().values().stream().flatMap(List::stream));
```

Comment:
Would a helper method like e.g. `Stream<String> RoutingProperties.allOutgoing()` be useful to make this easier to read?